Spatial.tscn

```diff
@@ -94,6 +94,17 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
+[node name="UnsavedDataIcon" type="TextureRect" parent="Control/GlobalMenuButton"]
+visible = false
+modulate = Color( 0.92549, 0, 0, 0.439216 )
+margin_top = 4.0
+margin_right = 25.0
+margin_bottom = 29.0
+texture = ExtResource( 3 )
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
```

Comment:
Do we need an entire separate node with its own offsets and details if all we are doing is modulating the color?
It should be decided if the correct behavior is color modulation, or a notification circle, or w/e.

This comment questioning how this feature should work should also go to show why checking if this node is visible or not to save/not-save a marker pack is unnecessary coupling. If all these changes went through and then later it was decided to just get rid of this node and modulate the base button then all of burrito would be unable to save for no reason.

Reply:
I needed a clear indicator when i was doing testing. After looking at it, I'm removing this node and just modulating the existing icon.